### PR TITLE
feat(Unplug): chatting without brain streaming

### PR DESCRIPTION
**Pull request by StepanLebedevTheodo**
# Description

Adds the possibility to chat with Quivr without selecting some brain

### CODE REVIEW

File: backend/core/routes/chat_routes.py [hunk 1]
```diff
@@ -190,17 +188,26 @@ async def create_question_handler(
...
-
-        gpt_answer_generator = OpenAIBrainPicking(
-            chat_id=str(chat_id),
-            model=chat_question.model,
-            max_tokens=chat_question.max_tokens,
-            temperature=chat_question.temperature,
-            brain_id=str(brain_id),
-            user_openai_api_key=current_user.user_openai_api_key,  # pyright: ignore reportPrivateUsage=none
-        )
+        # if not brain_id:
```
Comment (gozineb) [inline backend/core/routes/chat_routes.py:191]: Remove comments lines 191 and 191

File: backend/core/routes/chat_routes.py [hunk 2]
```diff
@@ -259,18 +266,28 @@ async def create_stream_question_handler(
...
-        gpt_answer_generator = OpenAIBrainPicking(
-            chat_id=str(chat_id),
-            model=(brain_details or chat_question).model if current_user.user_openai_api_key else "gpt-3.5-turbo",
-            max_tokens=(brain_details or chat_question).max_tokens if current_user.user_openai_api_key else 0,
-            temperature=(brain_details or chat_question).temperature if current_user.user_openai_api_key else 256,
-            brain_id=str(brain_id),
-            user_openai_api_key=current_user.user_openai_api_key,  # pyright: ignore reportPrivateUsage=none
-            streaming=True,
-        )
+        # if not brain_id:
```
Comment (gozineb) [inline backend/core/routes/chat_routes.py:269]: same comment

File: frontend/app/chat/[chatId]/hooks/useChat.ts [hunk 3]
```diff
@@ -45,14 +45,14 @@ export const useChat = () => {
         setChatId(currentChatId);
       }
 
-      if (currentBrain?.id === undefined) {
-        publish({
-          variant: "danger",
-          text: t("missing_brain"),
-        });
+      // if (currentBrain?.id === undefined) {
```
Comment (gozineb) [inline frontend/app/chat/[chatId]/hooks/useChat.ts:48]: remove comments

File: frontend/app/chat/[chatId]/hooks/useQuestion.ts [hunk 4]
```diff
@@ -59,9 +59,9 @@ export const useQuestion = (): UseChatService => {
     chatId: string,
     chatQuestion: ChatQuestion
   ): Promise<void> => {
-    if (currentBrain?.id === undefined) {
-      throw new Error(t("noCurrentBrain", { ns: "chat" }));
-    }
+    // if (currentBrain?.id === undefined) {
```
Comment (gozineb) [inline frontend/app/chat/[chatId]/hooks/useQuestion.ts:62]: remove

**Review by gozineb — COMMENTED** [review]
Remove the comments

**Comment by StepanLebedevTheodo** [comment]
> Remove the comments
Done

**Review by gozineb — APPROVED** [review]

**Pull request merged**